### PR TITLE
Us103342/polymer3/background type

### PR DESCRIPTION
**Pull request by johngwilkinson**

### CODE REVIEW

File: sass/page.scss [hunk 1]
```diff
@@ -61,6 +61,10 @@ body[unresolved] {
 	padding-bottom: 10px;
 }
 
+.d2l-page-dashboard {
+	background-color: #f6f7f8;
```
Comment (jwong-d2l) [inline sass/page.scss]: I'm not sure if your designer has talked to anyone about this, but I think as a whole they wanted to move to a named colour, specifically regolith.

Comment (johngwilkinson) [reply sass/page.scss]: Regolith seems to be #f9fafb, which doesn't match what's on the homepage now or the design I was given (both have #f6f7f8). I'll double-check with Fiona.

Comment (dlockhart) [reply sass/page.scss]: When this came up last time (with Joseph), the designers involved did express an interest in changing it. No matter what they decide, at least this way it's in one place.

Comment (johngwilkinson) [reply sass/page.scss]: Updated to use regolith

**Pull request merged**